src/components/forms/NumberField/NumberField.js

```diff
@@ -0,0 +1,56 @@
+import {
+  FormFieldDescription,
+  FormLabel,
+  Paragraph,
+  Textbox,
+  FormField as UtrechtFormField,
+} from '@utrecht/component-library-react';
+import classNames from 'classnames';
+import {Field} from 'formik';
+import PropTypes from 'prop-types';
+import React, {useRef} from 'react';
+
+import {getBEMClassName} from 'utils';
+
+export const NumberField = ({
+  name = '',
+  label = '',
+  isRequired = false,
+  description = '',
+  id = '',
+  disabled = false,
+  invalid = false,
+}) => {
+  const labelClassName = getBEMClassName('label', [isRequired && 'required'].filter(Boolean));
+
+  const inputProps = {
+    name,
+    as: Textbox,
+    id,
+    disabled,
+    invalid,
+    min: 0,
+    step: 1,
```

Comment:
these should not be hardcoded, but be props of the `NumberField` itself. You can then add two stories - one without `min` / `step`  specified -> assert that you can enter negative decimal values in the input and a second story with `min: 0` and `step: 1` specified -> assert that you can not enter negative or decimal values